nixos/tests/gphoto2/custom-group-name.nix

```diff
@@ -0,0 +1,22 @@
+import ../make-test-python.nix (_:
+let
+  machineName = "machine";
+in
+{
+  name = "gphoto2-with-custom-group-name";
+
+  nodes."${machineName}".programs.gphoto2 = {
+    enable = true;
+    group = "my-group";
+  };
+
+  testScript = ''
+    ${machineName}.start()
+
+    ${machineName}.succeed("gphoto2 --version")
+    ${machineName}.succeed("getent group my-group")
+
+    # Make sure it doesn't also create the default group
+    ${machineName}.fail("getent group camera")
```

Comment:
Running the program with --version is supposed to be done in passthru with the helper. Checking if the group is created doesn't add much value especially if we need to start an entire VM for that. We should do a proper integration test if the program works as intended or drop this.

Reply:
Wouldn't that require simulating an actual camera device? That sounds really difficult. The value of the current test is that it does verify that the change does what it says on the tin.